src/constans/ApiPaths.ts

```diff
@@ -0,0 +1,2 @@
+export const RANDOM_CHUCK_JOKE_PATH = '/random'
+export const CATEGORIES_PATH ='/categories'
```

Comment:
i think it would be better if you created one object instead of exporting separate constants:
` const PATHS={
    RANDOM_CHUCK_JOKE_PATH: '/random',
    CATEGORIES_PATH: '/categories'
    }`